seeds/park-seeds.js

```diff
@@ -42,21 +42,125 @@ const generateSqlArray = (parkData) => {
   let formattedSql = [];
   let parks = parkData.parks;
   for (let x = 0; x < parks.length; x++) {
-    formattedSql.push({id: x + 1, park_id: parks[x].id})
+    formattedSql.push({

```

Comment:
Commented this out. If you run http://localhost:3001/api/parks in insomnia you can see the IDs incrementing